samples/CosmosDBIntegration/CosmosDbIntegrationTutorial.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial: Setup Redis Triggers for Azure Function in Visual Studio
+Article 08/08/2023 Nadia Bugarin, Riley MacDonald, Phoebe Owusu
+
+In this tutorial, you learn how to:
+1. Install packages on an existing Azure function
+2. Use a Redis trigger with your Azure Function
+3. Deploy an Azure Function
+
+
+## Prerequisites
```

Comment:
I don't think all these are prerequisites. Some of this is just useful additional information. Some of it is just your list of Sample code. I would think the sample code is the story so you would essentially be implementing as you go. Or, are you creating additional Azure Functions themselves?